File: app/src/main/java/com/example/customseekbar/customseekbar/SeekBarState.java
package com.example.customseekbar.customseekbar;

public class SeekBarState {
    public String indicatorText;
    public int value; //now progress value
    public boolean isMin;
    public boolean isMax;

    @Override
    public String toString() {
        return "indicatorText: " + indicatorText + " ,isMin: " + isMin + " ,isMax: " + isMax;
    }
}

File: app/src/main/java/com/example/customseekbar/customseekbar/SeekBarState.kt
package com.example.customseekbar.customseekbar

class SeekBarState {
    var indicatorText: String? = null
    @JvmField
    var value //now progress value
            = 0
    var isMin = false
    var isMax = false
    override fun toString(): String {
        return "indicatorText: $indicatorText ,isMin: $isMin ,isMax: $isMax"
    }
}